.github/workflows/autobuild.yml

```diff
@@ -120,15 +120,15 @@ jobs:
              target_os:             macos
              building_on_os:        macos-10.15
              cmd1_prebuild:         "./autobuild/mac/artifacts/autobuild_mac_1_prepare.sh 5.15.2"
-             cmd2_build:            "./autobuild/mac/artifacts/autobuild_mac_2_build.sh"
+             cmd2_build:            "./autobuild/mac/artifacts/autobuild_mac_2_build.sh sign_if_possible"
```

Comment:
Although we’re using GH actions now, wouldn’t it be better to use/expand the -s option we have for deploy_mac.sh:
`-s if_possible` or `-s <cert>`? 

For reason see other comments.

Reply:
Sadly, we need to support the old build on qt 5.9. This means the config has to come from the build script like this

Reply:
Could you please elaborate? 

1. Is signing impossible for the old build?
2. How can we add new options ("jackonmac") if needed?

Reply:
No signing for old version, correct. 
For new options, we add them as needed.

Reply:
Ok.